.github/CONTRIBUTING.md

```diff
@@ -46,12 +47,12 @@ These tests are run both with the latest available version of `Nextflow` and als
 
 ## Patch
 
-: warning: Only in the unlikely and regretful event of a release happening with a bug.
+:warning: Only in the unlikely and regretful event of a release happening with a bug.
 
 * On your own fork, make a new branch `patch` based on `upstream/master`.
 * Fix the bug, and bump version (X.Y.Z+1).
 * A PR should be made on `master` from patch to directly this particular bug.
 
 ## Getting help
 
-For further information/help, please consult the [nf-core/eager documentation](https://nf-co.re/nf-core/eager/docs) and don't hesitate to get in touch on the nf-core Slack [#eager](https://nfcore.slack.com/channels/eager) channel ([join our Slack here](https://nf-co.re/join/slack)).
+For further information/help, please consult the [nf-core/eager documentation](https://nf-co.re/eager/docs) and don't hesitate to get in touch on the nf-core Slack [#eager](https://nfcore.slack.com/channels/eager) channel ([join our Slack here](https://nf-co.re/join/slack)).
```

Comment:
this link https://nf-co.re/eager/docs is not working anymore.
We should either get it fixed in the website to resolve to https://nf-co.re/eager/latest/usage or use https://nf-co.re/eager/latest/usage here or actually use the https://nf-co.re/eager/2.2.0/usage here since it'll be this release

Reply:
No change on this one yet

Reply:
Maybe worth mentioning in the TEMPLATE actually

Reply:
Will poin to latest, as this will always be latest release so we can assume would be pretty close to the one most people are using.